definitions/SpellLevels.dbd

```diff
@@ -6,8 +6,6 @@ int BaseLevel
 int MaxLevel
 int SpellLevel
 int MaxPassiveAuraLevel
-int LearnLevel // Player level thisis learned at.
```

Comment:
Note that this is an "official" name, coming from some Lua docs iirc.

Reply:
It's used in the same places in client as BaseLevel, and we have that name from old leaks